### DISCUSSION TITLE
Sonarr, Prowlarr and Transmission cannot communicate anymore after using Gluetun

### DISCUSSION THREAD

**Discussion by RobertVoorn1977** [qdm12/gluetun#2307] [Q&A]
I have a setup where one machine has prowlarr, sonarr running and another machine where transmission is running. This works fine without gluetun. When I link Gluetun to Sonarr, Prowlarr cannot access Sonarr anymore and Sonarr cannot reach the Transmission on the LAN anymore. this makes the whole setup useless and could use some help.

I added the environment setting 'FIREWALL_OUTBOUND_SUBNETS=192.168.1.0/24' to the Glueton docker hoping to get the Transmission connection working, but that is unfortunately not the case. When I use bash in Gluetun to ping Transmission that works fine, but from the Sonarr container transmission cannot be pinged and I get a 'unreachable network' error.

I have no idea why Prowlarr cannot reach Sonarr anymore as this is on the same machine and network and through the webbrowser, Sonarr is accessible.

**Comment by PrinceFrosty** [comment]
Have you pointed Prowlarr at the Gluetun container with Sonarrs port? Sounds like you might've not set `network_mode: "service:gluetun"` or something similar. It could also be DNS, have you set `DOT: off` or set static docker network ips? I'd suggest putting Prowlarr, Sonarr, and any other *Arrs in the same container as Gluetun and simply point them at `localhost:<port>`. I'm not sure about your Transmission problem, is it the correct subnet? Are you using docker overlays?

**Reply by RobertVoorn1977** [reply]
Hi, thanks for the reply. I currently do not have the possibility to check these things, but will try it tomorrow and will let you know. There are some things that I know I have not done and some things that are new to me and I have to look into.

**Reply by RobertVoorn1977** [reply]
Hi, I took me a while (things kept coming up in the mean time) but I have managed to fix this problem using the suggestions provided. Thanks for the help.
